apps/src/componentLibrary/dropdown/simpleDropdown/SimpleDropdown.story.tsx

```diff
@@ -92,6 +92,51 @@ DisabledDropdown.args = {
   size: 'm',
 };
 
+export const WithErrorDropdown = SingleTemplate.bind({});
+DisabledDropdown.args = {
```

Comment:
Thanks a lot for adding all of this stories!
Please note that they won't work untill you replace `DisabledDropdown.args` with `WithErrorDropdown.args`, etc for other new stories. Since right now code only updates DisabledDropdown story

Reply:
oof! thanks for catching this 😅